BUILD.bazel

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@com_github_bazelbuild_buildtools//buildifier:def.bzl", "buildifier")
+
+# Bazel buildifier: auto formats bazel files.
+buildifier(
```

Comment:
I wonder why buildifier is defined in BUILD.bazel, while the other go tools (that I assume buildifier depends on) are in WORKSPACE?

Reply:
That's right. buildifier dependencies added to WORKSPACE, but it also needs a build rule in root BUILD file. It complains if there is no BUILD file at root dir.